x/evm/types/statedb_test.go

```diff
@@ -60,3 +67,34 @@ func (suite *StateDBTestSuite) TestBloomFilter() {
 	suite.Require().True(ethtypes.BloomLookup(bloomFilter, contractAddress))
 	suite.Require().False(ethtypes.BloomLookup(bloomFilter, ethcmn.BigToAddress(big.NewInt(2))))
 }
+
+func (suite *StateDBTestSuite) TestStateDBWithContext() {
```

Comment:
you can define the `stateDB` as a field for the testing suite (i.e on `SetupTest`) and remove this test entirely 👍 

```go
suite.stateDB = suite.app.EvmKeeper.CommitStateDB.WithContext(suite.ctx)
```

Reply:
i believe i did this correctly, a quick check to the changes would be appreciated